tests/test.module-items.js

```diff
@@ -15,10 +15,102 @@ test("Assignment Items work", async t => {
       Selector(".ExpandCollapseList-item:nth-of-type(1) svg[name='IconAssignment']").exists
     )
     .ok()
-    .expect(
+    .click(
       Selector('.ExpandCollapseList-item:nth-of-type(1) a').withText(
         'First Module Assignment 1'
-      ).exists
+      )
     )
+    .expect(Selector('h1').withText('First Module Assignment 1').exists)
     .ok()
 });
+
+test("Quiz Items work", async t => {
+  const itemCss = '.ExpandCollapseList-item:nth-of-type(2)'
+  const title = `First Module Quiz 1`
+  await t
+    .expect(Selector(`${itemCss} svg[name='IconQuiz']`).exists)
+    .ok()
+    .expect(
+      Selector(
+        `${itemCss} div.ExpandCollapseList-item-details`
+      ).withText(`1 Questions`).exists)
+    .ok()
+    .click(Selector(`${itemCss} a`).withText(title))
+    .expect(Selector('h1').withText(title).exists)
+    .ok()
+});
+
+test("File Items work", async t => {
+  const itemCss = '.ExpandCollapseList-item:nth-of-type(3)'
+  const title = `341540_440x350.jpg`
```

Comment:
if theres nothing to interpolate should't we use double/single quote instead of back-tick?